src/hotspot/share/gc/shenandoah/shenandoah_globals.hpp

```diff
@@ -339,26 +335,16 @@
           "reserve/waste is incorrect, at the risk that application "       \
           "runs out of memory too early.")                                  \
                                                                             \
-  product(uintx, ShenandoahOldEvacReserve, 2, EXPERIMENTAL,                 \
-          "How much of old-generation heap to reserve for old-generation "  \
-          "evacuations.  Larger values allow GC to evacuate more live "     \
-          "old-generation objects on every cycle, while potentially "       \
-          "creating greater impact on the cadence at which the young- "     \
-          "generation allocation pool is replenished.  During mixed "       \
-          "evacuations, the bound on amount of old-generation heap "        \
-          "regions included in the collecdtion set is the smaller "         \
-          "of the quantities specified by this parameter and the "          \
-          "size of ShenandoahEvacReserve as adjusted by the value of "      \
-          "ShenandoahOldEvacRatioPercent.  In percents of total "           \
-          "old-generation heap size.")                                      \
-          range(1,100)                                                      \
-                                                                            \
-  product(uintx, ShenandoahOldEvacRatioPercent, 12, EXPERIMENTAL,           \
+  product(uintx, ShenandoahOldEvacRatioPercent, 75, EXPERIMENTAL,           \
           "The maximum proportion of evacuation from old-gen memory, as "   \
-          "a percent ratio.  The default value 12 denotes that no more "    \
-          "than one eighth (12%) of the collection set evacuation "         \
-          "workload may be comprised of old-gen heap regions.  A larger "   \
-          "value allows a smaller number of mixed evacuations to process "  \
+          "a percent ratio.  The default value 75 denotes that no more "    \
+          "than 75% of the collection set evacuation "                      \
```

Comment:
Is the underlying quantity the number of bytes eligible for evacuation or the number of regions eligible for evacuation? Perhaps make that more clear in the description string.